sycl/tools/sycl-ls/sycl-ls.cpp

```diff
@@ -69,6 +69,17 @@ static void printDeviceInfo(const device &Device, bool Verbose,
     std::cout << Prepend << "Name       : " << DeviceName << std::endl;
     std::cout << Prepend << "Vendor     : " << DeviceVendor << std::endl;
     std::cout << Prepend << "Driver     : " << DeviceDriverVersion << std::endl;
+
+    std::cout << Prepend << "Aspects    :";
+#define __SYCL_ASPECT(ASPECT, ID)                                              \
+  try {                                                                        \
+    if (Device.has(aspect::ASPECT))                                            \
+      std::cout << " " << #ASPECT;                                             \
+  } catch (...) {                                                              \
+  }
+
+#include <sycl/info/aspects.def>
```

Comment:
Should we also print the deprecated ones (aspects_deprecated.def)?

Reply:
I don't think so. After all, deprecated are (normally?) just aliases, so don't have any extra information.